calico/src/main/scala/calico/html/HtmlTag.scala

```diff
@@ -31,7 +31,7 @@ final class HtmlTag[F[_], E] private[calico] (name: String, void: Boolean)(using
   def apply[M](modifier: M)(using M: Modifier[F, E, M]): Resource[F, E] =
     build.toResource.flatTap(M.modify(modifier, _))
 
-  def apply[M](mkModifier: E => M)(using M: Modifier[F, E, M]): Resource[F, E] =
+  def withSelf[M](mkModifier: E => M)(using M: Modifier[F, E, M]): Resource[F, E] =
```

Comment:
Not to second guess 😅 maybe another option is `fromSelf`. But `withSelf` is good. I think 😅